src/test/java/uk/ac/ebi/embl/api/validation/submission/SubmissionValidationPlanTest.java

```diff
@@ -479,4 +481,25 @@ public void testGenomeSubmissionwitFastawithValidChromosomeListMultipleTimesInPa
 
 		CompletableFuture.allOf(futs.toArray(new CompletableFuture[futs.size()])).join();
 	}
+
+	@Test
+	public void testCovid19GenomeSize() throws IOException, ValidationEngineException {
```

Comment:
- Totally fine to have two integration tests for covid: one that passed the length validation and another that does not.
- Other than this I would add specific tests for the genome length for the different configuration (e.g. contig only etc.).